### ISSUE TITLE
Inaudible behaviour minimum too high?

### ISSUE THREAD

**Issue by GWRon** [jarikomppa/soloud#240]
Inaudible behaviour is cutting my sounds.

In my little 2D game I have an elevator which moves away from the player. Instead of using the 3D audio capabilities I use some simple volume and pan-adjustments to make it "positional" (it has to work with other sound engines as well).
The further the elevator goes the more silent his driving noise becomes. On each floor the elevator stops it plays an open sound ... a bing sound and a close sound. If effective volumes now change because player moves, it happens that the sound goes below 0.01f a bit - and then back again.

Both volumes were pretty well hearable in other mixers/soundengines. Here it leads to some sounds stopping here and there and then continuing if volume is big enough.
It is not "gradient" as I would assume.

Long story short summary:
Maybe make the lower barrier adjustable instead of fixed 0.01f. Eg. I replaced it with 0.001f and my sounds are playing back perfectly again.

I know there is the inaudibleBehaviour thing - but of course somewhen there is a "audible minimum" reached - it is just not the same for everyone and every "noisy" situation. 

soloud_core_3d.cpp:
```C
                if (vi->mOverallVolume < 0.01f)
                {
                    // Inaudible.
                    vi->mFlags |= AudioSourceInstance::INAUDIBLE;

                    if (vi->mFlags & AudioSourceInstance::INAUDIBLE_KILL)
                    {
                        stopVoice(voices[i]);
                    }
                }

**Comment by jarikomppa** [comment]
Good point. 1% audio may still be quite audible. I'll make the default lower and consider making it configurable.
